### DISCUSSION TITLE
Ability to include parameters in redirect upon submission.

### DISCUSSION THREAD

**Discussion by djpbadenhorst** [docusealco/docuseal#330] [Ideas]
First of all thank you for the work done. I've found it to be a pleasure to set up and use.

Perhaps the function I am seeking is available in some other way, but I was hoping to utilize a customised completed_redirect_uri that would have the context of the submission. Of course there are several ways to do this, perhaps the simplest would be have the ability to include `submission_id` or an alternative identifier in the completed_redirect_uri. Perhaps as a templated variable or a query parameter?

That way the front-end of the redirect would be able to include a download link or information around the submission.

If an alternative approach is feasible, I am open to suggestions.

**Comment by omohokcoj** [comment]
@djpbadenhorst pass external_id=(random generated id) in the URL - and also pass it in the POST /submissions external_id submitter json param.
This way you'll be able to match the redirect link external_id param and the submitter external_id

**Reply by djpbadenhorst** [reply]
This should do the job. 
Thanks (especially within a 1 min response time).
I will explore this solution.
